Bottom: 41fafd48989de6bfdba6f02016d05ad6d47887ef
Top:    9b6a0102750ee9eb746b46fa32f49a1ea77c4ff6
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-15 16:13:12 +0800

Refresh of tun-introduce-tx-array-mode

---

diff --git a/include/linux/net.h b/include/linux/net.h
index d2f9830..b6b3843 100644
--- a/include/linux/net.h
+++ b/include/linux/net.h
@@ -185,7 +185,7 @@ struct proto_ops {
 	ssize_t 	(*splice_read)(struct socket *sock,  loff_t *ppos,
 				       struct pipe_inode_info *pipe, size_t len, unsigned int flags);
 	int		(*set_peek_off)(struct sock *sk, int val);
-	int		(*peek_len) (struct socket *sock);
+	int		(*peek_len)(struct socket *sock);
 };
 
 #define DECLARE_SOCKADDR(type, dst, src)	\
